CHANGELOG.rst

```diff
@@ -14,6 +20,12 @@ Bug fixes
   `~climpred.classes.PredictionEnsemble.bootstrap` accept ``dim`` as ``list``, ``set``,
    ``tuple`` or ``str`` (:issue:`519`, pr:`558`) `Aaron Spring`_.
 
+Internals/Minor Fixes
+---------------------
+- docstrings are now tested in GitHub actions continuous integration.
+  (:issue:`545`, :pr:`560`) `Aaron Spring`_.
+- GitHub actions cancels previous runs. (:pr:`560`) `Aaron Spring`_.
```

Comment:
Can you clarify what that means? Is it that if you add a new commit it doesn't run testing on the old ones anymore?

Reply:
correct.

Reply:
Great... see comment above. I'd change this to just say "Github actions now cancels previous commits, instead of running the full testing suite on every single commit."